lib/tests/test_rewrite.rs

```diff
@@ -97,6 +97,7 @@ fn test_rebase_descendants_sideways() {
             commit_b.id().clone() => hashset!{commit_f.id().clone()}
         },
         hashset! {},
+        Default::default(),
```

Comment:
What do you think about setting this by default and then having a `set_options()`? That way we don't need to pass `Default::default()` in these tests.